storage/filestore/dir.go

```diff
@@ -18,10 +18,12 @@ import (
 )
 
 const (
-	blobPermission = 0600
-	dirPermission  = 0700
+	blobPermission = 0655
+	dirPermission  = 0755
 )
 
+var pathEncoding = base32.NewEncoding("abcdefghijklmnopqrstuvwxyz234567").WithPadding('1')
```

Comment:
Quite interesting encoding. I can understand that using the small letters instead of the caps improves readability. But why the "1" padding?

Reply:
because I had to pick something, I also could use `_`... or other valid file character

Reply:
Also I could use no-padding and a prefix

Reply:
fixed